docs/CHANGELOG.asciidoc

```diff
@@ -72,6 +73,80 @@ This section summarizes the changes in each release.
 //=== Known Issues
 ////
 
+[[release-notes-6.3.1]]
+== {kib} 6.3.1
+
+[float]
+[[breaking-6.3.]]
+=== Breaking Changes
+
+See <<breaking-6.3.0, Breaking Changes in 6.3.0>>.
+
+[float]
+[[enhancement-6.3.1]]
+=== Enhancements
+Management::
+* [Fixes #19475] Replaces the `watch-type-select` directive in Watcher 
+with buttons
+{pull}19782[#19782]
++
+[role="screenshot"]
+image::images/management-watcher-2-buttons.png[]
+
+Monitoring::
+* Improves the experience of enabling monitoring collection when the page is set 
+to an absolute time range. {pull}19451[#19451]
+
+Operations::
+* Bumps node to version 8.11.3 {pull}19861[#19861]
```

Comment:
This PR isn't marked as a release note item.  Is it definitely one we want to include?